### ISSUE TITLE
[xunit-plugin] xunit-filename parameter doesn't work on 0.19.0

### ISSUE THREAD

**Issue by eplaut** [slash-testing/slash#374]
```
slash run -vvvv test.py --with-xunit --xunit-filename 'test.xml'
usage: slash run [-h] [-i] [-o PATH=VALUE] [--repeat-each NUM_TIMES] [-x]
                 [-k FILTER] [-f FILENAME] [--tb PARAM]
                 [--truncate-console-lines yes/no] [-l DIR] [-q] [-v] [--pdb]
slash run: error: Unknown flags: --xunit-filename
```

with 0.18.2 it works

**Comment by eplaut** [comment]
thanks!

**Comment by vmalloc** [comment]
Fixed in 0.19.1 (just released)

**Comment by vmalloc** [comment]
n/p sorry for the inconvenience
